packages/terra-table/src/Table.jsx

```diff
@@ -35,6 +37,8 @@ const TableConstants = {
 
 const ROW_SELECTION_COLUMN_ID = 'table-rowSelectionColumn';
 
+const DEBOUNCE_TIMER = 300;
```

Comment:
Do we want to debounce or throttle the resize logic?  With a debounce, you won't see any updates until you finish resizing.  Also, 300ms seems pretty high for a delay after the resize is completed.